src/modules/user/controllers/controllerRegister.js

```diff
@@ -54,17 +53,13 @@ export const handleRegister = async function (req, res) {
       }
     }
 
-    const result = await databasePrisma.user.create({
+    await databasePrisma.user.create({
```

Comment:
If you keep the data you can add it to the req to pass along reducing the need for further database queries
```suggestion
    const user = await databasePrisma.user.create({
```